### DISCUSSION TITLE
Welcome to scratchStuff Discussions!

### DISCUSSION THREAD

**Discussion by sakkammadam** [sakkammadam/scratchStuff#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hbaird7877** [comment]
I am a member of platoon #2, the group responsible for this repository in connection with CSE 687 at Syracuse University Online.

**Comment by richardwrq** [comment]
您的来信已收到。谢谢
